src/popup/router/components/buttons/BtnMain.vue

```diff
@@ -73,6 +73,7 @@ export default {
 
   @extend %face-sans-16-regular;
 
+  margin: 0 auto;
```

Comment:
It was `margin: 8px auto;` not sure shall we have it 8 or 0.

Reply:
In my opinion buttons should not have any margins on their own.
And also I really like the rule, that I red once, which says that the components should not position themselves (on their own). The parent should take care about placing the child elements in correct places. This is why I'm rather against adding x-axis margins as auto. The default behaviour of buttons in HTML is that they are inline-blocks so they align with the text.